bc_obps/service/data_access_service/operator_service.py

```diff
@@ -10,6 +10,10 @@ class OperatorDataAccessService:
     def get_operator_by_id(cls, operator_id: UUID) -> Operator:
         return Operator.objects.get(id=operator_id)
 
+    @classmethod
+    def get_all_operators(cls) -> QuerySet[Operator]:
+        return Operator.objects.all()
+
```

Comment:
for now I think we can assume that yes, Declined operators should be excluded. 

For Gabe's context - in Reg 1 internal users wanted the ability to review new operators that were created and approve/decline them. They've since decided they don't need to do that, so Reg 2 doesn't have the concept, but in our prod database we have a few operators that are Declined.

Reply:
actually just checked Metabase - we don't have any Declined operators in prod. But I guess it's theoretically possible that one could come along between now and 2025